listenbrainz/db/dump.py

```diff
@@ -128,7 +128,24 @@
         'recording_mbid',
         'match_type',
         'last_updated'
```

Comment:
The mbid_mapping table is missing a check_again field, can you please also add it here

Reply:
Done.